packages/lambdas/src/ihe-inbound-document-retrieval.ts

```diff
@@ -1,25 +1,20 @@
 import { inboundDocumentRetrievalReqSchema } from "@metriport/ihe-gateway-sdk";
 import * as Sentry from "@sentry/serverless";
 import { processInboundDocumentRetrieval } from "@metriport/core/external/carequality/dr/process-inbound-dr";
-import { APIGatewayProxyEvent } from "aws-lambda";
 
-export const handler = Sentry.AWSLambda.wrapHandler(async (event: APIGatewayProxyEvent) => {
-  if (!event.body) {
-    return buildResponse(400, { message: "The Request body is missing" });
-  }
-  const payload = JSON.parse(event.body);
-  const baseRequest = inboundDocumentRetrievalReqSchema.parse({
-    id: payload.id,
-    timestamp: payload.timestamp,
-    samlAttributes: payload.samlAttributes,
-    documentReference: payload.documentReference,
-  });
-  const result = await processInboundDocumentRetrieval(baseRequest);
+export const handler = Sentry.AWSLambda.wrapHandler(async (event: string) => {
+  console.log(`Running with: ${event}`);
+  if (!event) return buildResponse(400, { message: "The request is invalid" });
+
+  const baseRequest = inboundDocumentRetrievalReqSchema.safeParse(JSON.parse(event));
+  if (!baseRequest.success) return buildResponse(400, baseRequest.error);
+  const result = await processInboundDocumentRetrieval(baseRequest.data);
+  console.log(`Response: ${result}`);
   return buildResponse(200, result);
 });
 
 const buildResponse = (status: number, body?: unknown) => ({
   statusCode: status,
   headers: { "Content-Type": "application/json" },
-  body: JSON.stringify(body),
+  body: body,
```

Comment:
Why not stringifying anymore? It seems `body` is an object?

Reply:
Its just the way the inbound processor wanted things

Reply:
Can you be more specific, please? :)

We're returning a string payload on a HTTP response here, right? If we don't stringify the object, it will be something like `[Object]`, don't you agree?

I wonder how is this working.

Reply:
Ok so the AWS-Lambda-Invoke.js calls this on the response:

const responseString = invokeResponse.payload().asUtf8String();
const responseJson = JSON.parse(responseString);

So it converts the payload to a string and then parses it. They might both work, but I know that sending it without JSON.stringify works ...